package.json

```diff
@@ -301,7 +301,7 @@
     "precommit": "lint-staged",
     "preinstall": "echo $npm_execpath | grep -q yarn || echo '\\033[0;33mSorry, npm is not supported. Please use Yarn (https://yarnpkg.com/).\\033[0m'",
     "prettier": "prettier --write '{enterprise/,}frontend/**/*.{js,jsx,ts,tsx,css}'",
-    "eslint-fix": "yarn && eslint --fix --ext .js,.jsx,.ts,.tsx --rulesdir frontend/lint/eslint-rules {enterprise/,}frontend/{src,test}",
+    "eslint-fix": "yarn && eslint --fix --ext .js,.jsx,.ts,.tsx --rulesdir frontend/lint/eslint-rules frontend/{src,test} enterprise/frontend/src",
```

Comment:
This wasn't runnable before

Reply:
That sounds like a good idea, since I'm not totally sure why we need to have a separate confix for this since the fix part is essentially the same but with `--fix` append at the back of the eslint command.